docker-compose.yml

```diff
@@ -10,6 +10,19 @@ networks:
 
 services:
 
+  foundry:
+    image: ghcr.io/foundry-rs/foundry:latest
```

Comment:
last minor nitpick can we set here a specific release? eg foundy:x.y.z?
I find it dangerous to use :latest since that's not deterministic. If you pull today you will get one version but if you pull tomorrow you will get another one. At some point they might introduce breaking changes and it will break our setup.